### PR TITLE
bundle images into output

### LINKED ISSUE #1: Images not added to 'build' bundle
with the current configuration, the images are not available within the built bundle after running `yarn build` (even though it works fine via `yarn dev`) - was able to resolve this by moving the images from `static/images` into the recommended `public` folder and that 'just worked'

If we want to keep the images separate within the static folder will need to do some additional post-processing post-build
otherwise could just move to public

### PR DESCRIPTION
**Pull request by ikanexus**
fixes #1 

Copies the files from `static/images` after `yarn build` is run - tested locally and now after doing `yarn build && yarn preview`, the images are now visible

### CODE REVIEW

**Comment by ikanexus** [comment]
https://package-images.pestoverse-ikanexus.pages.dev/
still seems to be failing on cloudflare pages though - maybe related to https://github.com/nuxt/image/issues/858

**Comment by Tougrel** [comment]
We should move to the native `img` element for the time being that works and keep an eye on the issue. If it is fixed then we can move back to nuxt-image once again 🤔

File: pages/credits.vue [hunk 1]
```diff
@@ -15,9 +12,9 @@ import ikanexus from 'static/images/credits/ikanexus.webp';
             </div>
 
             <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
-                <CreditsUser name="Yuniiho" description="For the amazing emotes" :iconPath="yuniiho" />
-                <CreditsUser name="Tougrel" description="Website idea & maintainer" :iconPath="tougrel" />
-                <CreditsUser name="Ikanexus" description="Ideas" :iconPath="ikanexus" />
+                <span v-for="user in CREDIT_USERS" >
+                    <CreditsUser :name="user.name" :description="user.description" :iconPath="user.icon_path" />
```
Comment (Tougrel) [inline pages/credits.vue:16]: Move the `v-for` from the `span` element to the `CreditUser` component 👍

**Comment by Tougrel** [comment]
I think that's all we can do right now for this pull request. If you have more to add then go ahead and request a review once you finish @ikanexus

**Review by Tougrel — APPROVED** [review]
All good here 👍

**Pull request merged**